listenbrainz/webserver/static/css/recommendation-page.less

```
@@ -0,0 +1,87 @@
+
+
+#recommendations {
+    padding: 10px;
+
+    > .recommendation-card {
+      padding: 15px;
+      margin-bottom: 7px;
+      max-height: 72px;
+
+      &.current-recommendation {
+        background-color: rgba(217, 237, 247, 0.3) !important ;
+      }
+
+       div {
+         padding: 0px;
+       }
+
+      > .col-xs-9, .col-xs-12 {
+
+        > .col-xs-9, .col-xs-12 {
+          display: flex;
+
+          .track-details {
+            width: 100%;
+
+            p {
+              overflow: hidden;
+              text-overflow: ellipsis;
+              white-space:nowrap;
+              margin-bottom: 0px;
+            }
+          }
+        }
+
+      }
+
+      > .col-xs-3 {
+        display: table;
+        height: 40px;
+
+        > * {
+          display: table-cell;
+          vertical-align: middle;
+        }
+
+        a {
+          word-break: break-word;
+        }
+
+        .recommendation-controls {
+
+          > svg:hover {
+            cursor: pointer;
+          }
+
+          .smile, .laugh, .angry, .disappoint, .meh {
+              font-size: 18px;
+              margin: 0% 7%;
+          }
+
+          .dropdown-menu {
+            min-width: 400px;
+            vertical-align: middle;
+            height: 40px;
```

This looks really wide to me. I'd suggest this to size to the contents:
```suggestion
            display: flex;
```